src/tasks/vesting_pool.ts

```diff
@@ -63,23 +67,59 @@ task("show_vestings", "Prints vesting details")
         let symbol = ""
         try {
             const tokenAddress = await vestingPool.token()
-            const token = await hre.ethers.getContractAt("SafeToken", tokenAddress) 
+            const token = await hre.ethers.getContractAt("SafeToken", tokenAddress)
             decimals = await token.decimals()
             symbol = await token.symbol()
         } catch {
             console.warn("Could not load token info!")
         }
-        const inputs: { 
+        const inputs: {
             vestingId: string | undefined,
         }[] = await readCsv(taskArgs.csv)
         for (const input of inputs) {
             if (input.vestingId) {
-                await printVestingDetails(vestingPool, input.vestingId, { decimals, symbol })
+                await loadVestingDetails(vestingPool, input.vestingId, { decimals, symbol })
             }
             console.log("")
         }
     });
 
+task("list_vestings", "Prints all vesting details")
+    .addParam("pool", "Vesting pool which should be queried", nameToAddress("Investor Vestings"), types.string, true)
+    .addParam("export", "If specified instead of printing the data will be exported as a json file for the transaction builder", undefined, types.string, true)
```

Comment:
When I run with `export` param I still get output in the logs. Also, I see that data is exported as csv and not JSON. Is it expected?